app/femr/ui/controllers/TriageController.java

```diff
@@ -240,6 +246,23 @@ public Result indexPost(int id) {
 		 if (viewModel.getWeeksPregnant() != null) { /*Sam Zanni*/
             newVitals.put("weeksPregnant", viewModel.getWeeksPregnant().floatValue());
         }
+
+        Calendar cal = Calendar.getInstance();
+        cal.setTime(viewModel.getAge());
+        int myAge = cal.get(Calendar.YEAR);
+        String classif = viewModel.getAgeClassification();
+
+        if (myAge <= 1951 && classif != null && classif != "elder")
```

Comment:
Checking a String for null does not take into consideration the empty string. Consider using the string untility (StringUtils.isNotNullOrWhiteSpace()) instead!